src/Powershell/NewInfrastructureFromTemplate.cs

```diff
@@ -20,10 +20,16 @@ public class NewInfrastructureFromTemplate : BaseCmdlet<NewInfrastructureFromTem
         )]
         public string TemplateName { get; set; }
 
+        [Parameter(HelpMessage = Resources.INFRASTRUCTURE_FORCE_HELP)]
+        public SwitchParameter Force;
+
+        [Parameter(HelpMessage = Resources.INFRASTRUCTURE_CREATION_SECONDS_TIMEOUT_HELP)]
+        public int? SecondsTimeout { get; set; }

```

Comment:
This parameter name is kind of weird. Why not just "Timeout" and having the description referring the units?